audit.py

```diff
@@ -834,15 +968,12 @@ def convert_alert_instance(instance):
             "end_column": instance.location.start_column,
         },
         "analysis_key": instance.analysis_key,
-        "message": instance.message.get('text')
+        "message": instance.message.get("text")
     }
     return converted
 
 
-JSON_DATETIME_FORMAT = "%Y-%m-%dT%H:%M:%S"
-
-
-def convert_alert(alert, verbose):
+def convert_alert(alert: CodeScanAlert, verbose: bool) -> REPORT_ITEMS:
```

Comment:
Why is report items all caps? User defined types should be camel case I think.

Reply:
Old habits fade slowly ;-)